server/app.js

```diff
@@ -7,25 +7,59 @@ const db = require('../db');
 app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
-
 app.use(express.static('./public'));
 
-//handle request for a certain seller
+//handle post requests for new seller
+app.post('/api/seller/', (req, res) => {
+
+  db.postSellerData(req.params, (err, results) => {
+    if (err) {
+      console.log('error occured posting seller info', err);
+      res.status(404).send('Error occured posting seller info');
+    }
+    res.status(200).send(results);
+  });
+
+});
+
+//handle get requests for specific seller
 app.get('/api/seller/:sellerID', (req, res) => {
 
   db.getSellerData(req.params.sellerID, (err, results) => {
     if (err) {
       console.log('error occured getting seller info', err);
       res.status(404).send('Error occured getting seller info');
     }
-
     res.status(200).send(results);
+  });
 
+});
+
+//handle put requests for specifc seller
+app.put('/api/seller/:sellerID', (req, res) => {
+
+  db.putSellerData(req.params.sellerID, (err, results) => {
```

Comment:
I think you may have forgot to pass in 'values' that your putSellerData function takes in